responsible-ai-collaborative/aiid
# Retrieve the backup file list from Cloudflare R2

This closes https://github.com/responsible-ai-collaborative/aiid/issues/2374

## Steps to test
- Go to https://deploy-preview-2375--staging-aiid.netlify.app/
- Click on footer link "Download Complete Database"
- Check if the list of files are displayed, ordered by date
- Try to download some snapshots and check if the links are pointing to R2 URL
